setup.py

```diff
@@ -10,6 +10,8 @@ def readme():
 requirements = []
 with open('requirements.txt') as f:
     for line in f.read().splitlines():
+        if line.startswith('# extras'):
```

Comment:
not sure if it'd be better to do this or create another requirements file for the extras